lib/output.js

```diff
@@ -603,6 +604,14 @@ function tile (options) {
         throw is.invalidParameterError('angle', 'positive/negative multiple of 90', options.angle);
       }
     }
+    // Background colour
+    if (is.defined(options.background)) {
```

Comment:
The `_setBackgroundColourOption` function already contains validation logic so we should be able to use a single, conditionless call to `this._setBackgroundColourOption(options.background)` here.

Reply:
OK I've tried that but test coverage still seems to fail 🤷‍♂️

Reply:
It should be possible to remove the `if (is.defined(...` check too. (I've just spotted a call to `_setBackgroundColourOption` in `lib/resize.js` also contains this unnecessarily-repeated logic, which I'll remove separately.)

Reply:
OK great, all checks now pass.